OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -161,7 +161,7 @@ public override IEnumerable<Target> GetTargets(Actor self)
 			return Target.None;
 		}
 
-		public override bool Cancel(Actor self)
+		public override bool Cancel(Actor self, bool keepQueue = false)
```

Comment:
I imagine that this has to be intentional, since you're skipping it for all the Move activities, but it seems wrong to ignore it.